Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2050,6 +2050,10 @@ - (void)_didFinishScrolling:(UIScrollView *)scrollView
     if (![self usesStandardContentView])
         return;
 
+#if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+    _isScrollingWithOverlayRegion = false;
```

Comment:
Very minor nit — `YES` / `NO` are normally used with `BOOL` (although `true` / `false` can effectively be used on all relevant platforms)

```suggestion
    _isScrollingWithOverlayRegion = NO;
```

Ditto w.r.t. the code in `handleScrollUpdate:` below that sets `_isScrollingWithOverlayRegion`